### PR TITLE
add tensorboard

### PR DESCRIPTION
**Pull request by cjcchen**
create a new PR for tensorboard support

### CODE REVIEW

File: rllab/misc/logger.py [hunk 1]
```diff
@@ -76,6 +79,18 @@ def remove_tabular_output(file_name):
         _tabular_header_written.remove(_tabular_fds[file_name])
     _remove_output(file_name, _tabular_outputs, _tabular_fds)
 
+def set_tensorboard_dir(dir_name):
+    global _tensorboard_writer
+    if dir_name is None or len(dir_name)==0:
```
Comment (ryanjulian) [inline rllab/misc/logger.py]:
```python
if dir_name:
```

File: rllab/misc/logger.py [hunk 2]
```diff
@@ -76,6 +79,18 @@ def remove_tabular_output(file_name):
...
+        if _tensorboard_writer is not None:
```
Comment (ryanjulian) [inline rllab/misc/logger.py]:
```python
if _tensorboard_writer:
```

File: rllab/misc/logger.py [hunk 3]
```diff
@@ -185,6 +203,21 @@ def refresh(self):
 
 table_printer = TerminalTablePrinter()
 
+def dump_tensorboard(*args, **kwargs):
+    if len(_tabular)>0 and _tensorboard_writer is not None:
```
Comment (ryanjulian) [inline rllab/misc/logger.py]:
```python
if len(_tabular) > 0 and _tensorboard_writer:
```

File: rllab/misc/logger.py [hunk 4]
```diff
@@ -185,6 +203,21 @@ def refresh(self):
...
+        tabular_dict = dict(_tabular)
+        if _tensorboard_step_key is not None and _tensorboard_step_key in tabular_dict:
```
Comment (ryanjulian) [inline rllab/misc/logger.py]:
```python
if _tensorboard_step_key and _tensorboard_step_key in tabular_dict:
```

**Comment by ryanjulian** [comment]
Looks good. Correct the minor comments (remember `None` and empty strings are falsy in Python!) and we can merge.

File: rllab/misc/logger.py (hunk 1 above)
Comment (cjcchen) [reply rllab/misc/logger.py]: Done

File: rllab/misc/logger.py (hunk 2 above)
Comment (cjcchen) [reply rllab/misc/logger.py]: Done

File: rllab/misc/logger.py (hunk 3 above)
Comment (cjcchen) [reply rllab/misc/logger.py]: Done

File: rllab/misc/logger.py (hunk 4 above)
Comment (cjcchen) [reply rllab/misc/logger.py]: Done

**Comment by ryanjulian** [comment]
CI is still failing :/

**Comment by cjcchen** [comment]
I have re-formated the code.

**Review by ryanjulian — DISMISSED** [review]

File: environment.yml [hunk 5]
```diff
@@ -48,6 +48,7 @@ dependencies:
         - git+https://github.com/Theano/Theano.git@adfe319ce6b781083d8dc3200fb4481b00853791#egg=Theano
         - git+https://github.com/neocxi/Lasagne.git@484866cf8b38d878e92d521be445968531646bb8#egg=Lasagne
         - git+https://github.com/plotly/plotly.py.git@2594076e29584ede2d09f2aa40a8a195b3f3fc66#egg=plotly
+        - git+https://github.com/deepmind/dm_control.git
```
Comment (ryanjulian) [inline environment.yml]: oops missed this. this dependency is unrelated to your change. please remove.

Comment (cjcchen) [reply environment.yml]: done

**Review by ryanjulian — APPROVED** [review]

**Comment by ryanjulian** [comment]
Fixes #10
